addons/l10n_nl/models/account_chart_template.py

```diff
@@ -23,3 +23,12 @@ def _prepare_transfer_account_for_direct_creation(self, name, company):
             res.setdefault('tag_ids', [])
             res['tag_ids'].append((4, xml_id))
         return res
+
+    @api.model
+    def _create_liquidity_journal_suspense_account(self, company, code_digits):
+        res = super(AccountChartTemplate, self)._create_liquidity_journal_suspense_account(company, code_digits)
+        if company.account_fiscal_country_id.code == 'NL':
+            res['tag_ids'] = [self.env.ref('l10n_nl.account_tag_25').id]
+            # Temporary solution - the account type should be changed for the whole account module
+            res['user_type_id'] = self.env.ref('account.data_account_type_current_assets').id
```

Comment:
For me, if we plan to change that for everyone, just do it directly instead of merging a "temporary solution" for a single localization.
Remove this line and let's do a commit in 14.0 to change this account type.

Reply:
OK I'll remove this line, but the change is already being implemented by someone else.

Reply:
@Shawcker What do you mean? By who? Where?